### PR TITLE
[CICD][SIMS #188]Feature/backup strategy 188

### PR DESCRIPTION
**Pull request by andrewsignori-aot**
- Created the backup structure to backup Postgres DBs and Mongo DB following the instructions from https://github.com/BCDevOps/backup-container
- Adapt the structure from the above documentation to our CICD.
- Changed the container sources to use the BC Gov artifactory,

- Deployed to our DEV env.
image

The below databases are configured to have the backup executed.
- postgres=patroni-master:5432/SIMSDB
- postgres=patroni-master:5432/sfasdb
- postgres=patroni-master:5432/ffa-camunda
- postgres=patroni-master:5432/ffa-api
- mongo=mongo:27017/formio

The backups will happen at 1 AM and they will be verified by 4 AM.

TODO
- Update the DEV Ops doc and test further the verification/restore.

### CODE REVIEW

File: devops/openshift/database-backup/backup.conf [hunk 1]
```diff
@@ -0,0 +1,12 @@
+postgres=patroni-master:5432/SIMSDB
+postgres=patroni-master:5432/sfasdb
+postgres=patroni-master:5432/ffa-camunda
+postgres=patroni-master:5432/ffa-api
+postgres=patroni-master:5432/ffa-analytics
+postgres=patroni-master:5432/ffa-session
+mongo=mongo:27017/formio
```
Comment (guru-aot) [inline devops/openshift/database-backup/backup.conf:7]: for now can we remove the analytics and session db backup?

**Review by guru-aot — APPROVED** [review]
Please remove the unwanted backup db's comment.
Otherwise
LGTM, great work. Now we will have DB backups :)

File: devops/openshift/database-backup/backup.conf (hunk 1 above)
Comment (andrewsignori-aot) [reply devops/openshift/database-backup/backup.conf:5]: Done!

**Review by ann-aot — APPROVED** [review]
Good Job 👍

**Review by abschwenker — APPROVED** [review]

**Pull request merged**